@navikt/core/react/tsconfig.build.json

```diff
@@ -7,5 +7,11 @@
     "declaration": true,
     "sourceMap": true
   },
-  "include": ["./src"]
+  "include": ["./src"],
+  "exclude": [
+    "./src/**/*.test.ts",
+    "./src/**/*.test.tsx",
+    "./src/**/*.test-d.ts",
+    "./src/**/*.stories.tsx"
+  ]
```

Comment:
This was added to fix these errors:
```
Error: [@navikt/ds-react]: [1] src/modal/types.test-d.ts(4,1): error TS2582: Cannot find name 'test'. Do you need to install type definitions for a test runner? Try `npm i --save-dev @types/jest` or `npm i --save-dev @types/mocha`.

[0] src/form/file-upload/utils/validate-files.test.ts(29,5): error TS2304: Cannot find name 'expect'. // multiple instances, just an example

src/tabs/Tabs.stories.tsx(7,1): error TS4082: Default export of the module has or is using private name 'TabsComponent'. // multiple instances, just an example
```

This could also be fixed by adding `"types": ["vitest/globals"]` to compilerOptions, but maybe it's a good idea to exclude these files anyways?